core/logger/audit/audit_logger.go

```diff
@@ -0,0 +1,215 @@
+package audit
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/smartcontractkit/chainlink/core/logger"
+
+	"github.com/pkg/errors"
+)
+
+const AUDIT_LOGS_CAPACITY = 2048
```

Comment:
Can drop the redundant `audit` prefix. 
Does this need to be exported?
Let's use mixed caps too: https://go.dev/doc/effective_go#MixedCaps
Maybe something like this? :shrug:
```suggestion
const bufferCapacity = 2048
```